python/cudf/cudf/pylibcudf_tests/test_column_from_device.py

```diff
@@ -35,17 +36,39 @@ def valid_type(request):
     return request.param
 
 
+class DataBuffer:
```

Comment:
Are we allowed to (in the tests) depend on cupy, then we could just use `cp.asarray()` in the test?

Reply:
Was discussing this with @lithomas1 , the issue with cupy is the missing support for datelike types

```
>>> cp.asarray([1,2,3], dtype='datetime64[ns]')
ValueError: Unsupported dtype datetime64[ns]
```

Reply:
Ah, ok